docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
+11. `package.json`
+12. `README.md`
+13. `yarn.lock`
 
-You should shortly see some text, close to the bottom, that says `The development server is listening at:` [http://localhost:8000](http://localhost:8000). Open that address in your
-browser and...
+Now that our site is up and running (at http://localhost:8000/), and we have been introduced to the files and directories that are powering it, let’s start exploring the starter code. (*Reminder, if you're not able to see your site running locally, make sure you've started running the development server with `gatsby develop`!*)
 
-![Gatsby.js hello world](hello-world.png)
+## Meet pages: The starter homepage
 
-Yeah! It's working!!!
+Open up the `/src` directory in your code editor.
 
-Too cool 😎
+Inside are three more directories -- `/components`, `/layouts`, and `/pages`. Open the file at `/src/pages/index.js`. The code in this file creates a component that contains the level-one header, two paragraphs, and link you see in the main content area of the starter homepage.
 
-Gatsby's development server is a "hot reloading" server, meaning any change you
-make to your React.js page components (and later we'll learn, your data files)
-will be immediately visible and/or loaded in the browser.
+### ✋ Make changes to the page markup
+1. Change some of the words, and edit the code on line 5 to add a border to the main div (see below), and save the file:
 
-This is huge because it makes development so much faster and fun.
 
-Let's try it.
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_E618395E4841D470F48D0AACF52A3EBAEA74345E0FB4AC285CBC937727755E20_1522460558137_gatsby1.gif)
 
-You'll need code editing software for the next part of this tutorial.
-[VS Code](https://code.visualstudio.com/) is a nice one. With your code editor,
-open the folder on your computer called "tutorial-part-one," which was
-automatically created in the location you chose when you ran the `gatsby new` terminal command above.
 
-Once you've opened the "tutorial-part-one" folder in your code editing
-software, it's time to edit your website. You'll see groups of directories and files; find the file in this location: `src/pages/index.js`. Once you open that file, try changing "Hello
-world!" in the page component to "Hello Gatsby!". Once you save that change, the text in your browser
-should change within a second (tip: you will always need to save changes before they appear in your browser).
+If your windows are side-by-side, you can see that your code and content changes are reflected almost instantly in the browser, once you save the file.
 
-Try some other tricks, like the ones below:
+### 💡 Hot reloading
+Gatsby uses "hot reloading" to speed up your development process. Essentially, when you're running a Gatsby development server, the Gatsby site files are being "watched" in the background -- any time you save changes to a file, your changes will be immediately reflected in the browser, without refreshing the page!
 
```

Comment:
Eventually (not necessary now), this would be a fun place to teach about the advantages of hot reloading, link to a case study, a blogpost, a quote, or just a pullout teaching moment

Reply:
Actually, even now it could be more focused on "why" hot reloading is handy.